package.json

```diff
@@ -10,14 +10,15 @@
     "url": "https://github.com/ipython/ipywidgets.git"
   },
   "scripts": {
+    "postinstall": "npm run bower",
     "build": "npm run css && npm run buildtests",
     "bower": "bower install --config.interactive=false",
     "precss": "rimraf ./ipywidgets/static/widgets/css/",
     "css": "node ipywidgets/build_css.js",
     "prebuildtests": "rimraf ./ipywidgets/tests/bin/tests",
     "buildtests": "tsc ./ipywidgets/tests/**/*.ts --outDir ./ipywidgets/tests/bin -d -m commonjs -t ES5",
-    "prepublish": "npm run bower && npm run build",
-    "publish": "npm run publish:pypi",
```

Comment:
Hm, I opened this PR because `npm install` failed with `no such command: bower`, but I re-ran it now after a clean and it's okay. Not sure what I had messed up.